framework/IO/TOutputWriter.php

```diff
@@ -12,23 +12,28 @@
 /**
  * TOutputWriter class.
  *
- * TOutputWriter extends TTextWriter to fwrite the buffer to STDOUT
+ * TOutputWriter extends TTextWriter to fwrite the buffer to "Output"
  * when {@see flush}ed.  This allows for testing of the Shell output.
  *
  * @author Brad Anderson <belisoful@icloud.com>
  * @since 4.2.0
  */
 class TOutputWriter extends TTextWriter
 {
+	/** @const The file path to open a data stream to Output. */
+	public const OUTPUT_URI = 'php://output';
+
+	/** @const The type of stream for Output. */
+	public const OUTPUT_TYPE = 'Output';
+
 	/**
 	 * Flushes the content that has been written.
 	 * @return string the content being flushed
 	 */
 	public function flush()
 	{
 		$str = parent::flush();
-		fwrite(STDOUT, $str);
-		flush();
+		echo $str;
```

Comment:
Is this intentional? the docblock still refers to "fwrite".
Also, afaik `STDOUT` bypassed the output buffer (`ob_start()`), while `echo` doesn't

Reply:
The original TOutputWriter was incorrectly writing to STDOUT rather than "Output".  The new TStdOutWriter is the direct to STDOUT class.  This was originally my error in TOutputWriter, my apologies.  I didn't know the precise details between STDOUT and "Output".

I've been studying PHP IO for a new Prado IO package and this is the correction.

For CLI, STDOUT and "Output" is the same (hence the confusion) but for PHP web pages, STDOUT is not defined and everything goes through "Output".   For web pages, this means that 'php://stdout' must be opened to have STDOUT.   Usually webpages don't need nor use STDOUT.  But for the Built-in PHP web server, we allow pages to open STDOUT to print to the CLI (eg, TStdOutLogRoute).

I'll make sure the doc is correct.

Reply:
The documentation is updated for clarity.